video-library/src/components/History/HistoryComp.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import "./History.css"
+import SideBar from "../SideBar/SideBar";
+import HistoryVideos from './HistoryVideos';
+function HistoryComp() {
+    return (
+        <div>
+        <div className="history-wrapper">
+        <div className="sidebar"> <SideBar/></div>
```

Comment:
I have a suggestion here, this Sidebar component is getting repeated in many components, you can do nested routing in such case, nested routing in router v6